Makefile

```diff
@@ -0,0 +1,40 @@
+# Build Info Variables
+FILE=./VERSION
```

Comment:
We should be consistent with the variable assignment style: `FILE=./VERSION` vs `FILE = ./VERSION`
Personally I feel `FILE = ./VERSION` is more readable.

Reply:
All assignments consistent with more readable format!